markdown/resourcemgmt/ConfigureResourceManagement.html.md.erb

```diff
@@ -100,22 +100,25 @@ In some cases, you may want to specify additional resource quotas on the query s
 
 The following configuration properties allow a user to control resource quotas without altering corresponding resource queues.
 
--   [hawq\_rm\_stmt\_vseg\_memory](../reference/guc/parameter_definitions.html)
 -   [hawq\_rm\_stmt\_nvseg](../reference/guc/parameter_definitions.html)
+-   [hawq\_rm\_stmt\_vseg\_memory](../reference/guc/parameter_definitions.html)
 
 However, the changed resource quota for the virtual segment cannot exceed the resource queue’s maximum capacity in HAWQ.
 
+***Note:*** The value of `hawq_rm_stmt_vseg_memory` depends on `hawq_rm_stmt_nvseg`, which defines the number of virtual segments used for the query. You must set `hawq_rm_stmt_nvseg` to a value greater than 0 before setting the memory quota for each segment with  `hawq_rm_stmt_vseg_memory`. After running a query or completing the session, the value of `hawq_rm_stmt_nvseg` should be reset to 0 to disable the statement level resource quota.
```

Comment:
Change "the value of `hawq_rm_stmt_nvseg` should be reset to 0 " to "reset the value of `hawq_rm_stmt_nvseg` to 0"